Sources/SourceControl/GitRepository.swift

```diff
@@ -112,6 +130,17 @@ public struct GitRepositoryProvider: RepositoryProvider, Cancellable {
         }
     }
 
+    @discardableResult
+    private func callGit(_ args: String...,
+                         environment: EnvironmentVariables = Git.environment,
+                         repository: RepositorySpecifier,
+                         failureMessage: String = "",
+                         progress: FetchProgress.Handler? = nil) throws -> String {
+        return try callGit(args.map { $0 }, environment: environment,
+                           repository: repository,
+                           failureMessage: failureMessage, progress: progress)
+    }
```

Comment:
formatting

Reply:
we have been asking folks to run `swiftformat` on changed code, see the "soundness" script